subsys/bluetooth/mesh/pb_gatt_srv.c

```diff
@@ -277,11 +277,12 @@ int bt_mesh_pb_gatt_srv_adv_start(void)
 	};
 	struct bt_data prov_sd[1];
 	size_t prov_sd_len;
-	int err;
+	int64_t timestamp = fast_adv_timestamp;
+	int64_t elapsed_time = k_uptime_delta(&timestamp);
```

Comment:
```suggestion
	uint8_t lefted_time = 60 - MIN(k_uptime_delta(&timestamp) / MSEC_PER_SEC, 60);
```

Reply:
In seconds, rather than milliseconds, it can ensure that there is at least 1 second to send a fast advertising.

Reply:
Unnecessary overcomplication with loss of time resolution since adv thread calculates the time in milliseconds.

Reply:
> In seconds, rather than milliseconds, it can ensure that there is at least 1 second to send a fast advertising.

Nop, due to resolution issue 1 second fast adv will work again only 110 ms means nothing again.